performance/example_test.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package performance_test
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/vmware/govmomi/performance"
+	"github.com/vmware/govmomi/simulator"
+	"github.com/vmware/govmomi/view"
+	"github.com/vmware/govmomi/vim25"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+func ExamplePerformance_ToMetricSeries() {
```

Comment:
CI failed here: https://travis-ci.org/vmware/govmomi/jobs/550296363
Needs to be `PerformanceManager` instead of just `Performance`

Reply:
Changed.

Reply:
Whoops, CI failed again, it should actually be `ExampleManager_ToMetricSeries`.  Has to be the same name as the type: https://github.com/vmware/govmomi/blob/2f2047990fde46ace7bb9f096fe2f74e4068c036/performance/manager.go#L36

Reply:
I didn't know that, thanks. Here we go now :)